### ISSUE TITLE
Add a config for Pokémons filter/notify

### ISSUE THREAD

**Issue by netzgeist** [AHAAAAAAA/PokemonGo-Map#1973]
I need a config for filtering and for getting notified about Pokémons. Prefferably as a list of IDs. 

Arguments:

* Manually adding alot of Pokémon via the frontend crunches it up to a point where you'd have to downscale everything. 
* The current process is also very time consuming. 
* Having a config I can switch between various behaviors conviniently. (Hunting rares vs farming them all)

Example: 

    run.py --filter filter.ini --notify notify.ini

**Comment by jelv** [comment]
We could also use a default with the old ignore (i) command:

`runserver.py -i Pidgey,Rattata,Weedle`

**Comment by daruda87** [comment]
The pokémons added to the front end, in which file are they saved? Just to be sure I won't delete them when I update my folder.

**Comment by invisiblek** [comment]
@daruda87 they're saved in your browser's [LocalStorage](http://www.w3schools.com/html/html5_webstorage.asp)

**Comment by brando56894** [comment]
Agreed, it would be useful to have a CLI switch that permanently blocks listed pokemon for the length of the run. My area is infested with Zubats and I have way too many of them so I always block them on the map.
